ui/webui/test/check-basic

```diff
@@ -213,6 +213,27 @@ class TestBasic(anacondalib.VirtInstallMachineCase):
 
         self._testLogReview(b, m, "critical-error")
 
+    def testJsErrorHandling(self):
+        b = self.browser
+        m = self.machine
+        i = Installer(b, m)
+
+        i.open()
+
+        b.wait_not_present("#critical-error-bz-report-modal.pf-c-modal-box")
+
+        with self.assertRaises(Error):
+            b.eval_js("a.b()")
+
+            b.wait_visible("#critical-error-bz-report-modal.pf-c-modal-box")
+            self._testLogReview(b, m, "critical-error")
+
+            b.assert_pixels(
```

Comment:
There was not screenshot generated or added by you. I think it's because the class it wrong.